### DISCUSSION TITLE
"Repository transfers not available for this account" error when attempting to transfer a repo

### DISCUSSION THREAD

**Discussion by supporting-Lines** [community/community#32273] [New to GitHub]
Hi,

I recently moved some repositories over to an organization that I created. I am the sole owner and admin of the organization. However, I realized that I do not need to have an organization, and want to move all the repos I transferred back to my account. However, every time I try that using the "Transfer Ownership" button in general settings, I get the following error: "Repository transfers not available for this account
". I've double-checked that I am the owner of the org. I've tried transferring ownership to a different account in the organization, and get the error "Repository transfers not available for this account and You can only transfer a repository from an organization to yourself at this time". This is fine, that's exactly what I want to do, but every time I try that I get an error. Please help!

Another piece of useful info is that there is a red banner when I visited my org that says "The ____ organization has been flagged. Because of that, your organization is hidden from the public. If you believe this is a mistake, [contact support](https://github.com/contact) to have your organization’s status reviewed."

Edit: I also have admin privileges on the repository itself

**Comment by radar-blip** [comment]
I would think all this is happening due to your organization being flagged. I would advise you contact support about it.
Which im sure, you already did, right?

**Reply by supporting-Lines** [reply]
Unfortunately not. I can't contact support because I am on a free plan

**Reply by airtower-luna** [reply]
You can contact support either way, here: https://support.github.com/request

**Comment by ghost** [comment]
Hi all. 

Something happened to me today, I don't know why I got this warning.

![image](https://user-images.githubusercontent.com/111701513/201234701-928064ec-2d3b-49c0-a2b6-f7355beafeab.png)

I've tried reaching out to the GitHub team, but I have no response.
